CSharp/ProgramRunner/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using InvertedIndexSearcher;
+using InvertedIndexSearcher.database;
+using InvertedIndexSearcher.database.dataproviders;
+using InvertedIndexSearcher.modifiers;
+using InvertedIndexSearcher.modifiers.filterers;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace ProgramRunner
+{
+    static class Program
+    {
+        private static IServiceProvider _serviceProvider;
+
+        static void Main()
+        {
+            CreateHost();
+            /*var dataProvider = (DefaultDataProvider) _serviceProvider.GetService(typeof(IDataProvider));
```

Comment:
Comment!

Reply:
for the sake of reading default files ( mentioned before as random text in phase1) we comment this 
but thanks for mentioning